src/components/coin-row/UnderlyingAssetCoinRow.js

```diff
@@ -0,0 +1,47 @@
+import React, { useCallback } from 'react';
+import { colors } from '../../styles';
+import { magicMemo } from '../../utils';
+import { ButtonPressAnimation } from '../animations';
+import { CoinIcon } from '../coin-icon';
+import { Centered, Row } from '../layout';
+import { Text } from '../text';
+
+const TopMoverCoinIconSize = 20;
+
+const UnderlyingAssetCoinRow = ({
+  address,
+  change,
+  isPositive,
+  name,
+  onPress,
+  price,
+  symbol,
+}) => {
+  const handlePress = useCallback(() => {
+    onPress?.({ address, change, name, price, symbol });
```

Comment:
```suggestion
    onPress({ address, change, name, price, symbol });
```
I think that everywhere `onPress` present